.gitignore

```diff
@@ -31,4 +31,13 @@ cpptests
 /pennylane_lightning/.vs
 /pennylane_lightning/*.pyd
 /pennylane_lightning/src/Kokkos/
-/pennylane_lightning/src/GBenchmarks/
\ No newline at end of file
+/pennylane_lightning/src/GBenchmarks/
+
+
+*sh
+/PennyLane_Lightning_GPU*
+/pennylane_lightning/core/src/simulators/lightning_gpu/MPIWorker.hpp
+/pennylane_lightning/core/src/simulators/lightning_gpu/utils/MPIManager.hpp
+/pennylane_lightning/core/src/simulators/lightning_gpu/utils/CSRMatrix.hpp
+/pennylane_lightning/core/src/simulators/lightning_gpu/*MPI.cpp
+/pennylane_lightning/core/src/simulators/lightning_gpu/*MPI.hpp
```

Comment:
Do we really want to exclude all of these?

Reply:
Just exclude for this PR.